CMakeLists.txt

```diff
@@ -33,7 +33,32 @@ dispatch_common_warnings()
 option(ENABLE_DISPATCH_INIT_CONSTRUCTOR "enable libdispatch_init as a constructor" ON)
 set(USE_LIBDISPATCH_INIT_CONSTRUCTOR ${ENABLE_DISPATCH_INIT_CONSTRUCTOR})
 
-# TODO(compnerd) swift options
+option(ENABLE_SWIFT "enable libdispatch swift overlay" OFF)
+if(ENABLE_SWIFFT)
```

Comment:
This looks like a typo. ENABLE_SWIFFT => ENABLE_SWIFT. I didn't read anything else though so don't take this as a "true" review.

Reply:
Nice catch.